compiler/src/iree/compiler/Codegen/LLVMGPU/TransformExtensions/LLVMGPUExtensions.cpp

```diff
@@ -30,6 +32,10 @@ using namespace mlir;
 using namespace mlir::iree_compiler::IREE;
 
 iree_compiler::IREE::transform_dialect::LLVMGPUExtensions::LLVMGPUExtensions() {
+  // CreateAsyncGroupsOp depends on the following two dialects.
+  declareGeneratedDialect<gpu::GPUDialect>();
```

Comment:
I actually didn't realize we had that ... I spent so much time connecting workarounds for exactly this in various undebuggable places in IREE ...
Thanks for using the right thing!